DESCRIPTION

```diff
@@ -25,7 +25,9 @@ Imports:
     reshape2,
     stats,
     utils,
-    rlang
+    rlang,
+    tidyr,
+    ggridges
```

Comment:
I'm always hesitant to add more dependencies, especially when only used in one or two places. Do you think __tidyr__ will be useful for other stuff going forward too? If so then I think it's fine to add the dependency now, but if not then is it possible to implement this without tidyr?

Reply:
I think I figured out a way to do it without tidyr.

Reply:
Thanks. I'm totally fine with using tidyr now if you think you'll probably want to use it a lot going forward. If not then I think the way you found of doing it without tidyr looks fine. Sorry if it was a pain to figure out.